cluster-provision/images/vm-image-builder/example/image-url-arm64

```diff
@@ -0,0 +1 @@
+https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/32/Cloud/aarch64/images/Fedora-Cloud-Base-32-1.6.aarch64.qcow2
```

Comment:
Can you update this to F35 at least? The same image as `fedora-kubeadm` & `fedora-with-test-tooling`

```
https://download.fedoraproject.org/pub/fedora/linux/releases/35/Cloud/aarch64/images/Fedora-Cloud-Base-35-1.2.aarch64.qcow2
```

Reply:
The x86_64 image (for this and `fedora-realtime` too) is Fedora 32, so I think it's better to keep architectures in sync even if that means starting off with very old content.

Ideally, a follow-up PR would bump things to something less ancient, such as Fedora 38, across the board.

Reply:
I will update the Fedora OS in a follow-up PR.